brickstrap-utils.sh

```diff
@@ -183,27 +196,25 @@ Do you have necessary binfmt packages installed?"
 #
 # Check whether commandline or environment based configuration of QEMU is
 # 'valid', i.e that it corresponds to an existing QEMU interpreter.
-# Returns 1 if no QEMU was set, to signal brp_determine_qemu
-# Fails if QEMU is explicitly misconfigured.
+# The variable 'BRP_BINFMT' is set to the passed QEMU interpreter if valid.
+# Returns 255 if no QEMU was configured. Fails if the QEMU settings are
+# misconfigured.
 #
 function brp_validate_qemu()
 {
     # check environment variable first, may have been set by config
     if [ -n "$QEMU_STATIC" -a -x "$QEMU_STATIC" ]; then
-        echo -n "$(readlink -f $QEMU_STATIC)"
+        BRP_BINFMT="$(readlink -f $QEMU_STATIC)"
```

Comment:
Furthermore, could we call `BRP_BINFMT` something like `BRP_SOURCE_QEMU` or `BRP_QEMU_SOURCE_PATH`?